index.bs

```diff
@@ -251,6 +253,25 @@ Their [=deserialization step=], given |serialized| and |value|, are:
 
 2. Set |value|'s underlying byte sequence to |serialized|.\[[ByteSequence]].
 
+<div algorithm="get stream">
+A {{Blob}} |blob| has an associated <dfn for=Blob>get stream</dfn> algorithm,
+which runs these steps:
+
+1. Let |stream| be the result of [=construct a ReadableStream object|constructing=] a
+   {{ReadableStream}} object.
+1. Run the following steps [=in parallel=]:
+  1. While not all bytes of |blob| have been read:
+    1. Let |bytes| be the byte sequence that results from reading a [=chunk=] from |blob|.
+    1. If a [=file read error=] occured while reading |bytes|, [$ReadableStream/error$]
+       |stream| with a [=failure reason=] and abort these steps.
+    1. [=ReadableStream/Enqueue=] |bytes| into |stream|.
```

Comment:
Readable streams, at least currently, take JS objects, not Infra things like byte sequences. For example https://fetch.spec.whatwg.org/#concept-read-all-bytes-from-readablestream assumes that the chunks are Uint8Arrays. So, if the goal is to integrate with that, you'd want to explicitly construct a Uint8Array.

Alternately, you could leave this as byte sequences as that's arguably a more correct long-term infrastructure choice. It's just the way that readable streams are defined right now that's making things difficult. I'm not sure what's best.